services/searchengine/bleveengine/bleve.go

```diff
@@ -231,6 +231,14 @@ func (b *BleveEngine) GetVersion() int {
 	return 0
 }
 
+func (b *BleveEngine) GetFullVersion() string {
```

Comment:
What was the reason for the `(*BleveEngine).GetFullVersion` vs `(*BleveEngine).GetVersion`?